packages/vats/test/bootstrapTests/supports.js

```diff
@@ -332,7 +341,8 @@ export const makeSwingsetTestKit = async (
 
   const storage = makeFakeStorageKit('bootstrapTests');
 
-  const marshal = makeMarshal();
+  const slotToVal = (_slotId, iface = 'Remotable') => Far(iface);
```

Comment:
This should not have worked, as it will cause a double prefixing. Do you know why it worked anyway? I would like to understand this before we proceed.

```suggestion
  const slotToVal = (_slotId, iface = 'Remotable') => Remotable(iface);
```

Reply:
Ah, I misunderstood the requirements that `Remotable()` enforces: I thought the iface must start with `Alleged:` (so how could the default of `Remotable` be accepted?), but now I see that it also accepts a plain string `"Remotable"`. I did indeed have a double prefixing at some point during testing (maybe when the default was `iface = "Alleged: Remotable"`, which in retrospect made even less sense).

Switching to `Remotable()` means whoever provides the serialized data must obey the same rule. I'll try it and see if anything breaks.